inc/class-users-importer.php

```diff
@@ -0,0 +1,137 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Bootstraps the Drupal 7 Importer plugin.
+ *
+ * Most? of the methods in this class are not type-hinted because the upstream
+ * files were not written as such, and I don't want us to accidentally
+ * break things.
+ */
+namespace Pragmatic\Drupal7_Importer;
+
+use HMCI\CLI;
+use HMCI\Iterator;
+use HMCI\Utils;
+use Pragmatic\Drupal7_Importer\Integration;
+
+/**
+ * Drupal 7 users importer.
+ */
+class Users_Importer extends Iterator\DB\Base {
+
+	/**
+	 * Constructor.
+	 *
+	 * @param array $args  Importer options.
+	 * @param string $type Is the iterator being used as a "validator" or an "importer"?
+	 */
+	public function __construct( $args = [], $type = 'importer' ) {
+
+		parent::__construct( $args, $type );
+		$this->args['debugger'] = 'error_log';
+	}
+
+	/**
+	 * Get importer description.
+	 *
+	 * @return string
+	 */
+	public static function get_description() {
+		return __( 'Drupal DB Importer description', 'pragmatic-drupal7-importer' );
+	}
+
+	/**
+	 * Get the total number of Drupal 7 users to migrate.
+	 *
+	 * @return int
+	 */
+	public function get_count() {
+
+		return Integration\get_drupal_users_count(
+			$this->db()
+		);
+	}
+
+	/**
+	 * Get Drupal entities.
+	 *
+	 * @param mixed $offset How many pages of items do we need to offset?
+	 * @param mixed $count  How many items do we fetch per query?
+	 *
+	 * @return array Drupal entities to migrate.
+	 */
+	public function get_items( $offset, $count ) {
+
+		if ( $count === '-1' ) {
+			$count = 100;
+		}
+
+		return Integration\get_drupal_users(
+			$this->db(),
+			(int) $count,
+			(int) $offset
+		);
+	}
+
+	/**
+	 * Transform the to-be-migrated user.
+	 *
+	 * @param array $item The import item.
+	 *
+	 * @return array The import item.
+	 */
+	public function parse_item( $item ) {
+
+		if ( ! $item ) {
+			return false;
+		}
+
+		$item['user_pass'] = wp_generate_password( 12, false );
+
+		/**
+		 * NOTE: Generate a random email address to avoid spamming real users during our tests.
+		 *
+		 * @todo remove for "actual" import.
+		 */
+		$user_data['user_email'] = sanitize_title( $item['user_login'] ) . '@example.local';
```

Comment:
Where is this variable used?

Reply:
Copypasta. 🤦‍♂ I just forgot to remove it.